src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
```

you should specify which properties you apply the transition to. in this case `all` is used which is not quite ok
```suggestion
    transition: transform 300ms, color 300ms;
```